### ISSUE TITLE
Pull request into homebrew official formulas

### ISSUE THREAD

**Issue by booi** [bls/homebrew-blackbox#1]
Hey, I was about to write the formula for this tool. Have you considered having this pulled into the official homebrew formulas?

**Comment by bls** [comment]
Nope, I'm basically done with it (don't need it any more personally). If
you want to fork / copy, do any fixups you feel neccessary and submit, go
right ahead! :)

Thanks & regards,

     Blair.

**Comment by bls** [comment]
Actually, if you want, I can transfer the repo to you?

**Comment by booi** [comment]
Sure. Either way I'll take care of it :).

**Comment by bls** [comment]
Cool!  :+1:  -> inbound to you

**Comment by kytwb** [comment]
Anybody following up on this?
